webapi/src/main/scala/org/knora/webapi/responders/ActorDeps.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.responders
+import akka.actor.ActorRef
+import akka.actor.ActorSystem
+import akka.util.Timeout
+import zio.ZIO
+import zio.ZLayer
+
+import scala.concurrent.ExecutionContext
+
+import org.knora.webapi.config.AppConfig
+import org.knora.webapi.core.AppRouter
+import org.knora.webapi.settings.KnoraDispatchers
+
+/**
+ * Class encapsulating all Akka dependencies necessary to interact with the [[org.knora.webapi.core.actors.RoutingActor]] aka. "appActor"
+ *
+ * When using this class in a service depending on the routing actor this will provide the necessary implicit dependencies for using the ask pattern
+ * whilst making the dependency explicit for ZIO layers.
+ *
+ * @example Usage in client code:
+ * {{{
+ * final case class YourService(actorDeps: ActorDeps){
+ *
+ *   // Put ExecutionContext and Timeout in implicit scope for the service
```

Comment:
```suggestion
```

This line can be probably removed as it seems to me not bringing much explanation.